src/app/server/JointFabricDatastore.cpp

```diff
@@ -627,7 +727,7 @@ CHIP_ERROR JointFabricDatastore::RemoveACLFromNode(uint16_t listId, NodeId nodeI
     {
         if (it->nodeID == nodeId)
         {
-            // TODO: mark as delete pending, update device, erase
+            it->statusEntry.state = Clusters::JointFabricDatastore::DatastoreStateEnum::kDeletePending;
```

Comment:
```suggestion
            // TODO: update device
            it->statusEntry.state = Clusters::JointFabricDatastore::DatastoreStateEnum::kDeletePending;
```

Reply:
Isn't "update device" essentially the same as committing a new entry into the vector?